scripts/west_commands/build.py

```diff
@@ -407,10 +412,7 @@ def _parse_test_item(self, test_item, board):
                         required_snippets.extend(arg_list)
                         continue
 
-                    if self.args.cmake_opts:
-                        self.args.cmake_opts.extend(args)
-                    else:
-                        self.args.cmake_opts = args
+                    self.args.cmake_opts.extend(args)
```

Comment:
Can't `self.args.cmake_opts` still be `None` here? (same for change further down the function)